### DISCUSSION TITLE
2023/01/01/installing-evdi-fedora37/

### DISCUSSION THREAD

**Discussion by jdeluyck** [jdeluyck/kcoreorg-website#135] [Blog Comments]
# 2023/01/01/installing-evdi-fedora37/
Installing Displaylink drivers on Fedora 37
https://kcore.org/2023/01/01/installing-evdi-fedora37/

**Comment by jdeluyck** [comment]
Originally posted by Daft Zero on 2023-03-17T00:44:24Z

----

hey you need to add an extra step after cloning the git repo it is necesary to perform checkout to this commit

git checkout f41389cb3617917fa5d00adf43b479c0e5656d1a

and run make

**Reply by jdeluyck** [reply]
This is no longer needed as this commit is part of the devel branch and part of the [upcoming release tagged as 1.13.0](https://github.com/DisplayLink/evdi/releases/tag/v1.13.0)

**Comment by masroorz** [comment]
Hi, 
This doesn't seem to work on Fedora 38 :(
